### ISSUE TITLE
Ruby on Rails - Asset pipeline problem

### ISSUE THREAD

**Issue by erdemgezer** [zurb/foundation-sites#7236]
When I try using the latest [foundation.min.css](http://foundation.zurb.com/sites/download.html/) in a Ruby on Rails project, asset pipeline gives the following error during precompile:

```
Sass::SyntaxError: Base-level rules cannot contain the parent-selector-referencing character '&'.
```

**Comment by timurkhafizov** [comment]
Same here.
I think the problem is here: `.media-object-section:last-child:not( + &:first-child)`
It is line `2229` in `foundation.css`

**Comment by erdemgezer** [comment]
I managed to get it working by loading the saas files with bower and leaving the compilation to rails. But it requires some workaround and doesn't feel right. Probably the best thing is to wait until the rails gem release.

**Comment by timurkhafizov** [comment]
I tried to install foundation-sites via `rails-assets` but somehow my markup totally broke after that.

**Comment by cavpollo** [comment]
+1

**Comment by gakimball** [comment]
Foundation 6.0.6 includes the fix that addresses this: b53e0b539e9d3c5002c445e73067fa62168d4f89
